### ISSUE TITLE
R Demo By Matt C (09/19)

### ISSUE THREAD

**Issue by anitabe404** [studydash/cards#416]
## Your Daily Accomplishment 🚀:
Spent an hour with Matt learning about R and RStudio. He gave me a walkthrough of the IDE along with places to find good blog posts and free books and info.

## References 🔗:
- [RPubs](https://rpubs.com/)
- [R-Bloggers](https://www.r-bloggers.com/)
- [Chris Albon: DS & ML](https://chrisalbon.com/)
- [R on Stackoverflow](https://stackoverflow.com/questions/tagged/r)

## Extra/Fun (Optional) 🎈🎉🥳:
Started watching season 2 of Lupin on Netflix.

**Comment by r002** [comment]
Very cool that you've been keeping up with Matt!  🥳

`R` is super great though!  Do you guys meet often or on a schedule?  If so, we could all get together on a group hangout sometime if you're keen.  Lmk!

**Comment by anitabe404** [comment]
> Do you guys meet often or on a schedule?

@r002 Not really. He emails me occasionally and it just so happened that we both had time Sunday. I accepted your friend request and messaged you in Discord. Maybe we can find a time when we can all meet. He seems interested in doing demos of stuff. I'm sure we could learn tons from seeing some of your work.

**Comment by r002** [comment]
@anitabe404 Gotcha, makes sense!  Yes, I've messaged you back in Discord as well! 🙂  I'd love if we all met though-- that sounds like a great plan.  Maybe sometime this weekend if you're free?  I'd love to show you guys the stuff I'm currently working on and also see what you guys have been up to as well!  Let's convene!  It shall be a meeting of the minds!! 😁
